### PR TITLE
First runnable build

### PR DESCRIPTION
**Pull request by DrexHD**
Today I have fixed some more issues and made the project runnable.

### CODE REVIEW

File: src/main/java/me/drex/itsours/claim/list/ClaimList.java [hunk 1]
```diff
@@ -88,8 +88,17 @@ private void addRegion(Region region, AbstractClaim claim) {
         return byOwner.get(owner) == null ? new ArrayList<>() : byOwner.get(owner);
     }
 
-    public List<AbstractClaim> get(ServerWorld world, BlockPos pos) {
-        return get(Region.get(pos.getX(), pos.getY())).stream().filter(abstractClaim -> abstractClaim.getWorld().equals(world)).collect(Collectors.toList());
+    public AbstractClaim get(ServerWorld world, BlockPos pos) {
```
Comment (ItsIlya) [inline src/main/java/me/drex/itsours/claim/list/ClaimList.java:91]: This has to be annotated with `@Nullable`

File: src/main/java/me/drex/itsours/claim/permission/PermissionManager.java [hunk 2]
```diff
@@ -19,21 +19,28 @@ public PermissionManager(CompoundTag tag) {
     }
 
     private void fromNBT(CompoundTag tag) {
```
Comment (ItsIlya) [inline src/main/java/me/drex/itsours/claim/permission/PermissionManager.java:21]: camelCase? `fromNbt`

File: src/main/java/me/drex/itsours/claim/permission/PermissionManager.java [hunk 3]
```diff
@@ -19,21 +19,28 @@ public PermissionManager(CompoundTag tag) {
...
+                        Role role = ItsOursMod.INSTANCE.getRoleManager().get(roleID);
+                        roleWeight.put(role, weight);
+                    });
+                    roles.put(UUID.fromString(uuid), roleWeight);
+                });
+            }
+        }
     }
 
     public CompoundTag toNBT() {
```
Comment (ItsIlya) [inline src/main/java/me/drex/itsours/claim/permission/PermissionManager.java:46]: Same as above

File: src/main/java/me/drex/itsours/claim/permission/PermissionManager.java (hunk 2 above)
Comment (DrexHD) [reply src/main/java/me/drex/itsours/claim/permission/PermissionManager.java:21]: nbt means **N**amed **B**inary **T**ag and therefor it should in my opinion stay liked that

File: src/main/java/me/drex/itsours/claim/list/ClaimList.java (hunk 1 above)
Comment (DrexHD) [reply src/main/java/me/drex/itsours/claim/list/ClaimList.java:91]: When I declared it with `@Nullable` the IDE auto imported `import com.sun.istack.internal.Nullable;`, which spit errors when I tried compiling

Comment (ItsIlya) [reply src/main/java/me/drex/itsours/claim/list/ClaimList.java:91]: Why don't you use Jetbrain's Annotations `org.jetbrains.annotations`?

File: src/main/java/me/drex/itsours/claim/permission/PermissionManager.java (hunk 2 above)
Comment (ItsIlya) [reply src/main/java/me/drex/itsours/claim/permission/PermissionManager.java:21]: Well I would like to follow Yarn's naming schemes and NBT is named Nbt

**Pull request merged**